app/data/password_blacklist/site_specific.txt

```diff
@@ -0,0 +1,17 @@
+digital
+marketplace
+digitalmarketplace
+digital-marketplace
+digital_marketplace
+digital marketplace
+government
+gcloud
+g-cloud
+g_cloud
+g cloud
+procurement
+opportunity
+opportunities
+requirement
+requirements
+commercial
```

Comment:
Do we want to bother with l33t versions of these too - e.g. `d1g1t4lm4rk3tpl4c3`? Or are those strong enough 😸